CHANGELOG/CHANGELOG-1.x.md

```diff
@@ -36,6 +36,8 @@
 * Optimize pipeline log style. [#802](https://github.com/erda-project/erda-ui/pull/802)
 * Optimize pipeline notification content. [#1189](https://github.com/erda-project/erda/pull/1189)
 * Optimize the execution logic of automation test cases. [#1103](https://github.com/erda-project/erda/pull/1103)
+* Support filtering test case executor by unassigned person in the test plan. [#732](https://github.com/erda-project/erda/pull/732)
+* Add precheck for front-end applicaiton packaging.
```

Comment:
Suggest to describe more clear as `precheck ths package-lock.json file's existence`

Reply:
Yeah it's more accurate

Reply:
@daskyrk Do you know what the PR refer to?

Reply:
Nope, it'a not in frontend repo.

Reply:
Suggest to search in erda issues and find out who is responsible for this job.